src/facade/conn_context.h

```diff
@@ -36,17 +36,10 @@ class ConnectionContext {
     return protocol_;
   }
 
-  SinkReplyBuilder* reply_builder() {
+  SinkReplyBuilder* reply_builder_old() {
```

Comment:
why do you called this _old?

Reply:
so that we won't use it. I will remove it in follow up PRs